src/pages/gravy.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import styled from "styled-components";
+
+import CloseButton from "../components/atoms/CloseButton";
+import Gravy from "../components/organisms/Gravy";
+import useInitialGlobalNavSplit from "../hooks/useInitialGlobalNavSplit";
+
+const Container = styled.div`
+  margin-top: 48px;
```

Comment:
Don't need this. It is adding a 48px header to the page. Can you check it against the live site to make sure it matches?

Reply:
Removed. Can't see a header on the live site.